sktime/forecasting/model_selection/tests/test_split.py

```diff
@@ -482,3 +483,69 @@ def _check_train_test_split_y(fh, split):
 
     cutoff = train.index[-1]
     np.testing.assert_array_equal(test.index, fh.to_absolute(cutoff).to_numpy())
+
+
+def test_split_series():
+    """Tests that split_series produces series in the split."""
+    y = _make_series()
+    cv = SlidingWindowSplitter()
+
+    for train, test in cv.split_series(y):
+        assert isinstance(train, pd.Series)
+        assert len(train) == 10
+        assert isinstance(test, pd.Series)
+        assert len(test) == 1
+
+
+def test_split_loc():
+    """Tests that split_loc produces loc indices for train and test."""
+    y = _make_series()
+    cv = SlidingWindowSplitter()
+
+    for train, test in cv.split_loc(y):
+        assert isinstance(train, pd.DatetimeIndex)
+        assert len(train) == 10
+        y.loc[train]
+        assert isinstance(test, pd.DatetimeIndex)
+        assert len(test) == 1
+        y.loc[test]
+
+
+def test_split_series_hier():
+    """Tests that split works with hierarchical data."""
+    hierarchy_levels = (2, 4)
+    n_instances = np.prod(hierarchy_levels)
+    n = 12
+    y = _make_hierarchical(
+        hierarchy_levels=hierarchy_levels, max_timepoints=n, min_timepoints=n
+    )
+    cv = SlidingWindowSplitter()
+
+    for train, test in cv.split(y):
+        assert isinstance(train, np.ndarray)
+        assert train.ndim == 1
+        assert train.dtype == np.int64
+        assert len(train) == 10 * n_instances
+        assert isinstance(test, np.ndarray)
+        assert test.ndim == 1
+        assert test.dtype == np.int64
+        assert len(test) == 1 * n_instances
+
+    for train, test in cv.split_loc(y):
+        assert isinstance(train, pd.MultiIndex)
+        assert len(train) == 10 * n_instances
+        y.loc[train]
```

Comment:
is this here just for debugging?

Reply:
no - this will raise an exception if `train` is not an index subset of `y.index`

Reply:
ok, I see. Is it possible to make it a bit more explicit statement, something like `assert x.issubset(y)`. sorry, I am away from python console to suggest the exact statement...

Reply:
done. Took me 10min to find this.